ProcessMaker/modeler
# FOUR-8437 add functionality to undo, redo controls

## Issue & Reproduction Steps

Expected behavior: 
Use the new undo/redo controls in order to have a better user experience

Actual behavior: 
These controls don't have any functionality

## Solution
- Implement the functionalities to the undo/redo controls

## How to Test
Test the steps above

1. Open a BPMN process
2. It should show the new undo/redo controls at the bottom-middle of the screen
3. Click over the Undo button then the process designer must go to the previous step
4. Click over the Redo button then the process designer must go to the next step

## Related Tickets & Packages
- [FOUR-8437](https://processmaker.atlassian.net/browse/FOUR-8437)

## Code Review Checklist
